CDS Maps/Individual Maps/CDS-to-FHIR/nNGM-CDS-to-FHIR-Sonstige Therapie.map

```diff
@@ -11,8 +11,8 @@ uses "http://hl7.org/fhir/StructureDefinition/Patient" as target
 group TransformBundleSonstigeTherapie(source operations: BackboneElement, target bundle: Bundle, target composition: Composition, target section: BackboneElement)
 {
     operations then CreateProcedureSonstigeTherapien(operations, bundle, composition, section);
-    operations then CreatePatientSonstigeTherapie(operations, bundle, composition, section);
-}
+    // operations then CreatePatientSonstigeTherapie(operations, bundle, composition, section);
```

Comment:
Everything related to the patient can be deleted

Reply:
410